applications/clover_network.md

```diff
@@ -0,0 +1,182 @@
+| 10. | Clover Wallet App | Android/iOS Native | Delivering initial build of Clover official wallet on both android/iOS |
+| 11. | Clover Wallet Chrome Extension| JavaScript/H5 | Delivering initial build of Chrome Extension of Clover web3 wallet |
+| 12. | Substrate module: Frontier-EVM pallet| Rust | Rolling out minor EVM adjustments with Clover chain |
+
+### February 2021 Milestones
+* **Estimated Duration:** 1 month
+* **FTE:**  11.5
+* **Costs:** 0.29 BTC
+
+| Number | Deliverable | Language/Framework | Specification |
+| ------------- | ------------- | ------------- | ------------- |
+| 1. | Documentation | Text | Covering 50% of the overall features on Clover Network |
+| 2. | Unit Test | TypeScript | The chain specific unit tests will cover 90% by the end of Feb. | 
+| 3. | Clover Chain | Rust | Implementing enterprise developers' achitecture with in the low level logic |
+| 4. | Clover Explorer | Vue/NodeJS/SpringCloud | Delivering the beta build of Clover cross chain explorer with support of BTC/ETH/DOT/CLV |  
+| 5. | Clover OS SDK | Native/JS | The first version of Clover OS SDK with Android/iOS version will be released by the end of Feb. |  
+| 6. | Clover Wallet App | Android/iOS Native | Integrating Clover OS on Clover Wallet App |  
+| 7. | Clover Wallet Chrome Extension | JavaScript/Vue | Finalizing Clover chrome extension and distribute it on google store |  
+| 8. | Tulip Editor | Vue/NodeJS/Solidity | Deliverig first POC version of Tulip editor |  
+| 9. | Clover Developer Portal | Vue/Java/NodeJS/PostgreSQL | Delivering basic functionalities of developer portal with account creation/upload/upgrade eAPP |
+| 10. | IPFS Integration | NodeJS/Go | Integrating IPFS node along with Clover node into single Docker file for distribution |
+
+### March 2021 Milestones
+* **Estimated Duration:** 1 month
+* **FTE:**  11.5
+* **Costs:** 0.29 BTC
+
+| Number | Deliverable | Language/Framework | Specification |
+| ------------- | ------------- | ------------- | ------------- |
+| 1. | Documentation | Text | Documentation will be ready at 1.0 version to cover 100% of the current features |
+| 2. | Tulip Editor | Vue/NodeJS/Solidity | Delivering alpha build for Tulip Editor with EVM support |
+| 3. | Clover Wallet App | Android/iOS Native | Releasing Clover wallet app on Google Play and Apple Store |  
+| 4. | Clover OS | Native/JS | Having Clover OS support for iOS/Android/Embedded by the end of Mar. |  
+| 5. | Clover Store | Vue/IPFS/Solidity/NodeJS | Delivering the official release of Clover Store. |  
+| 6. | Clover Developer Portal | Vue/Java/NodeJS/PostgreSQL | Rolling out Clover developer portal for the public with incentivize program |  
+| 7. | Clover Wallet Firefox/Safari Extension | Javascript/Vue | Delivering Clover Firefox/Safari extension |  
+| 8. | Clover Governance | Solidity/Rust | Releasing first build of Clover governance system |
+| 9. | Storage | NodeJS | Adding IPFS/AR/CRUST support as the Storage service |
```

Comment:
These milestones seem... extraordinarily ambitious. Are you certain you can complete them in one-month time periods?

Reply:
Yes, quite ambitious to deliver such amount of work, however we have to deliver as much as we can in shortest time possible. We really need to push our limits to hit early milestones which will leave us even more milestones.